service.go

```diff
@@ -240,6 +244,22 @@ func (svc *Service) createResponse(initialEvent *nostr.Event, content interface{
 	return resp, nil
 }
 
+func (svc *Service) GetMethods(app *App) []string {
+	appPermissions := []AppPermission{}
+	findPermissionsResult := svc.db.Find(&appPermissions, &AppPermission{
+		AppId: app.ID,
+	})
+	if findPermissionsResult.RowsAffected == 0 {
+		// No permissions created for this app. It can do anything
+		return strings.Split(NIP_47_CAPABILITIES, ",")
```

Comment:
going forward we might want to say that no permission means it can not do anything?

Reply:
@bumi that will break old connection strings, right? but I think it would be good - less complexity in the app.

I will create a separate issue for this.